index.html

```diff
@@ -11192,6 +11192,52 @@ <h2>A project for first-time contributions</h2>
                     </div>
                 </div>
                 <!-- ________ PR_Lowell's card END ________  -->
+
+                                <!-- ________ Garvit's card START ________  -->
```

Comment:
Thank you for your PR. You are almost there - just 1 step away.

As you can see here, your code is over-indented. It kind of messes with the structure of this HTML page, and we would like it to be consistent. Could you please fix the indentation to be in line with the other cards?